ReleaseProcessAutomation.Tests/Jira/JiraCredentialManagerTests.cs

```diff
@@ -0,0 +1,165 @@
+using System.Net;
+using AdysTech.CredentialManager;
```

Comment:
I think we are also missing the unit test from the previous project?

Reply:
I wouldn't know where they are as I think I already included all of them? I think the test s are just missing in general. I'm creating unit tests for `JiraEntrancePoint` and possibly `JiraCredentialManager`

Reply:
I just missed them I think, my bad :D